sdk/eventhub/Azure.ResourceManager.EventHubs/src/Azure.ResourceManager.EventHubs.csproj

```diff
@@ -9,4 +9,8 @@
     <PackageTags>azure;management;eventhub</PackageTags>
     <NoWarn>$(NoWarn);AZC0001;AZC0008</NoWarn>
   </PropertyGroup>
+
```

Comment:
The change looks fine but I don't understand why wasn't the EventHubs code generated/updated before?

Reply:
OKay, I figured out why, started a thread.

Reply:
I don't either, but it fails to build in master

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/19860 is the fix to include management projects back into all checks